swift-swiftLint.md

```diff
@@ -0,0 +1,131 @@
+* [CocoaPod](https://cocoapods.org)
+
+# Installation
+
+### Using [Homebrew](http://brew.sh/):
+
+```
+brew install swiftlint
+```
+
+### Using [CocoaPods](https://cocoapods.org):
+
+Simply add the following line to your Podfile:
+
+```ruby
+pod 'SwiftLint'
+```
+
+This will download the SwiftLint binaries and dependencies in `Pods/` during next
+`pod install` execution and will allow to invoke it via `${PODS_ROOT}/SwiftLint/swiftlint`
+in Script Build Phases.
+
+> This is the recommended way to install a specific version of SwiftLint since it supports
+installing a pinned version rather than simply the latest (which is the case with Homebrew).
+
+# Configuration
+Create a file `.swiftlint.yml` file in the root directory of your project.
+<br />Then add the below configuration in this file.
+
+```yaml
+disabled_rules: # rule identifiers to exclude from running
+  - vertical_whitespace
+  - line_length
+  - type_body_length
+  - colon
+  - comma
+opt_in_rules: # some rules are only opt-in
+  - control_statement
+  - empty_count
+  - trailing_newline
+  - convenience_type
+  - empty_string
+  - file_name
+  - toggle_bool
+  - force_unwrapping
+  - unused_private_declaration
+  - missing_docs
+  - unused_declaration
+  - unused_import
+  - indentation_width
+  - discouraged_optional_boolean
+  - fallthrough
+
+included: # paths to include during linting. `--path` is ignored if present.
+  - Project
+  - ProjectTests
+  - ProjectUITests
+  
+excluded: # paths to ignore during linting. Takes precedence over `included`.
+  - Pods
+  - Carthage
+  - Project/R.generated.swift
+
+# configurable rules can be customized from this configuration file
+# binary rules can set their severity level
+force_cast: warning # implicitly. Give warning only for force casting
+
+force_try:
+  severity: warning # explicitly. Give warning only for force try
+  
+line_length:
+  warning: 140
+  error: 160
+
+type_body_length:
+  - 300 # warning
+  - 400 # error
+
+# or they can set both explicitly
+file_length:
+  warning: 600
+  error: 800
+  
+indentation: 2
+
+large_tuple: # warn user when using 3 values in tuple, give error if there are 4
+   - 5
+   - 6
+   
+ missing_docs:
+  warning: 
+  - private
+  - open
+  - fileprivate
+  - public
+  - internal
+  
+# naming rules can set warnings/errors for min_length and max_length
+# additionally they can set excluded names
+type_name:
+  min_length: 3 # only warning
+  max_length: # warning and error
+    warning: 30
+    error: 35
+  excluded: iPhone # excluded via string
+reporter: "xcode" # reporter type (xcode, json, csv, checkstyle, codeclimate, junit, html, emoji, sonarqube, markdown, github-actions-logging)
+```
+
+# Usage
+If you've installed SwiftLint via CocoaPods, then 1. **Select Target** -> 2. **Build Phase** -> 3. **Click on +** -> 4. **Add New Run Script Phase** and <br /> copy & paste below script in Run Script
```

Comment:
Can we put the items 1, 2, 3, in new lines?

Reply:
updated